### PR TITLE
SIMPLE: added scripts for running test suite in quiet mode

### PR DESCRIPTION
**Pull request by jhsu802701**
The docker/qtest script makes it possible to omit the thousands of annoying and distracting deprecation warnings when running the test suite.

### CODE REVIEW

**Review by andrewpbrett — APPROVED** [review]

**Comment by andrewpbrett** [comment]
I verified that this suppresses the deprecation warnings as intended. I think it's safe to merge without a second review.

**Comment by sauloperez** [comment]
@jhsu802701 don't forget that those deprecation warnings need to be fixed eventually. Feel free to fix some :joy:

**Comment by jhsu802701** [comment]
> @jhsu802701 don't forget that those deprecation warnings need to be fixed eventually. Feel free to fix some joy

I fully agree, but this pull request is part of a compromise that we all can work with.

**Pull request merged**
